components/dropdown/Dropdown.module.css

```diff
@@ -1,7 +1,9 @@
 .dropdownContainer {
   display: flex;
-  width: 327px;
+  order: 2;
+  width: 100%;
   margin-bottom: 20px;
+  position: relative;
```

Comment:
<img width="477" alt="Screenshot 2023-08-29 at 21 46 44" src="https://github.com/luizfiorentino/fndr/assets/20372832/620a7959-5cab-4c93-88cd-6537d8aa3d30">

I added position relative to the dropdown

- The X button was absolutely positioned (good choice)
- But not relative to the dropdown, it was positioned relative to the whole page
- This makes the X button difficult to keep in the right spot (see example in screenshot) 
- Added position relative to the form, makes the X button relative to the input